packages/tradershub/src/features/cfd/screens/MT5TradeScreen/MT5TradeScreen.tsx

```diff
@@ -0,0 +1,153 @@
+
+    const marketType = getCFDState('marketType');
+    const platform = getCFDState('platform') ?? mt5Platform;
+
+    const platformToAccountsListMapper = useMemo(
+        () => ({
+            ctrader: ctraderAccountsList?.find(account => account.is_virtual === activeAccount?.is_virtual),
+            dxtrade: dxtradeAccountsList?.find(account => account.is_virtual === activeAccount?.is_virtual),
+            mt5: account,
+        }),
+        [ctraderAccountsList, dxtradeAccountsList, account, activeAccount?.is_virtual]
+    );
+
+    const details = platformToAccountsListMapper[platform as TPlatforms.All];
+
+    const loginId = useMemo(() => {
+        if (platform === mt5Platform) {
+            return (details as THooks.MT5AccountsList)?.display_login;
+        } else if (platform === dxtradePlatform) {
+            return (details as THooks.DxtradeAccountsList)?.account_id;
+        }
+        return (details as THooks.CtraderAccountsList)?.login;
+    }, [details, dxtradePlatform, mt5Platform, platform]);
+
+    return (
+        <div className='lg:w-[45vw] lg:min-w-[51.2rem] lg:max-w-[60rem] w-full min-w-full h-auto'>
+            <div className='flex flex-col p-1200 gap-800 border-b-100 border-system-light-secondary-background'>
+                <div className='flex items-center justify-between w-full'>
+                    <div className='flex items-center'>
+                        <div className='mr-400'>
+                            {platform === mt5Platform
+                                ? MarketTypeDetails[(marketType ?? 'all') as keyof typeof MarketTypeDetails].icon
+                                : PlatformDetails[platform as keyof typeof PlatformDetails].icon}
+                        </div>
+                        <div className='flex flex-col'>
+                            <div className='flex items-center flex-row gap-300'>
+                                <Text size='md'>
+                                    {platform === mt5Platform
+                                        ? MarketTypeDetails[(marketType ?? 'all') as keyof typeof MarketTypeDetails]
+                                              .title
+                                        : PlatformDetails[platform as keyof typeof PlatformDetails].title}{' '}
+                                    {!activeAccount?.is_virtual && details?.landing_company_short?.toUpperCase()}
+                                </Text>
+                            </div>
+                            <Text className='text-system-light-less-prominent-text' size='sm'>
+                                {loginId}
+                            </Text>
+                        </div>
+                    </div>
+                    <div className='flex flex-col items-end'>
+                        <Text bold>{details?.display_balance}</Text>
+                    </div>
+                </div>
+                <div className='flex flex-col gap-100'>
```

Comment:
```suggestion
                <div className='space-y-100'>
```

Reply:
@aizad-deriv suppose you are wrong, we need to use flex for space (https://tailwindcss.com/docs/space#add-horizontal-space-between-children)